api/app/signals/settings/development.py

```diff
@@ -5,7 +5,7 @@
 SECRET_KEY = 'insecure'
 DEBUG = True
 ALLOWED_HOSTS = ['*']
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_BACKEND = os.getenv('EMAIL_BACKEND', 'django.core.mail.backends.console.EmailBackend')
```

Comment:
This is only for development so no need to change this. The console EmailBackend will show the send mails in the logs. I would leave this as is. If you want to overwrite this for local development there is always the option of doing so in the `local.py` settings file.

Reply:
I am using MailHog in my local development workflow.

Reply:
Same comment. I think that in general we need to take a look at the setting files and make them more consistent. Not blocking for this PR.